reproman/distributions/tests/test_venv.py

```diff
@@ -62,8 +63,8 @@ def venv_test_dir():
     if ssp:
         # The testing environment supports --system_site_packages.
         pip2 = op.join("venv-nonlocal", "bin", "pip")
-        runner.run(["virtualenv", "--python", PY_VERSION,
-                    "--system-site-packages", "venv-nonlocal"])
+        runner.run(venv_cmd + ["--python", PY_VERSION,
```

Comment:
I'm confused by the lack of coverage here.  There is a dedicated job that sets should sets `REPROMAN_TESTS_ASSUME_SSP=1`, so this should be triggered.  I expected to have to do more to make the --system-site-packages job work (if even feasible), so it not being in effect would explain why it "just worked".